src/Etebase.test.ts

```diff
@@ -1700,3 +1700,52 @@ it.skip("Login and password change", async () => {
   const etebase4 = await Etebase.Account.login(USER2.email, USER2.password, testApiBase);
   await etebase4.logout();
 }, 30000);
+
+describe("Chunking files", () => {
+  it("Duplicate Chunks", async () => {
+    const collectionManager = etebase.getCollectionManager();
+    const col = await collectionManager.create(colType, {}, "");
+
+    const buf = randomBytesDeterministic(10 * 1024, new Uint8Array(32)); // 10kb of psuedorandom data
+    const content = JSON.stringify([buf, buf, buf, buf]);
+
+    await col.setContent(content);
+
+    await collectionManager.transaction(col);
+    const decryptedContent = await col.getContent();
+
+    const out = toString(decryptedContent);
+    expect(out).toEqual(content);
```

Comment:
```suggestion
  it("Duplicate Chunks", async () => {
    const collectionManager = etebase.getCollectionManager();
    const col = await collectionManager.create(colType, {}, "");

    const buf = randomBytesDeterministic(10 * 1024, new Uint8Array(32)); // 10kb of psuedorandom data
    const content = new Uint8Array([...buf, ...buf, ...buf, ...buf]);

    await col.setContent(content);

    await collectionManager.transaction(col);
    const out = await col.getContent();

    expect(out).toEqual(content);
```

Cleaner way of doing this test. Can you change the other ones accordingly?